packaging/aws-cloudformation/firecamp-ecs.template

```diff
@@ -96,7 +96,6 @@
           "Fn::Join":[
             "-",
             [
-              "firecamp",
```

Comment:
Thanks for catching it! Could you please fix the firecamp-swarm.template as well?